src/main.js

```diff
@@ -25,7 +25,7 @@ Vue.filter('stat', function (value) {
 })
 Vue.filter('aggressive', function (value) {
 	if (value == undefined) return 0;
-	if (value >= 1000) {
+	if (value >= 1000 && !rootGetters["settings/showFullValues"]) {
 		value = Math.min(Math.floor(value / 1000), 9);
 		return `>${value}k`;
 	}
```

Comment:
Can't comment down there, but on line 32 you'll want to return `value.toLocaleString()` so you get the commas in there (this piece of code hasn't needed commas yet).